apps/desktop/src/main/lib/agent-setup/templates/codex-wrapper-exec.template.ps1

```
@@ -0,0 +1,101 @@
+	$logPath = $env:CODEX_TUI_SESSION_LOG_PATH
+
+	$WatcherJob = Start-Job -ScriptBlock {
+		param($log, $notify)
+
+		function Send-HookEvent([string]$notifyScript, [string]$eventName) {
+			$payload = ('{{"hook_event_name":"{0}"}}' -f $eventName)
+			try {
+				powershell.exe -NoProfile -ExecutionPolicy Bypass -File $notifyScript $payload | Out-Null
+			} catch {
+				# Silent — lifecycle notifications must not block codex.
+			}
```

PSScriptAnalyzer (1.25.0)

[warning] Empty catch block is used. Please use Write-Error or throw statements in catch blocks.

(PSAvoidUsingEmptyCatchBlock)